shared/ios/Keybase/AppDelegate.m

```diff
@@ -76,6 +78,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
   UIViewController *rootViewController = [UIViewController new];
   rootViewController.view = rootView;
+  UIDropInteraction *udi = [[UIDropInteraction alloc] initWithDelegate:self];
+  udi.allowsSimultaneousDropSessions = YES;
+  rootView.interactions = @[udi];
```

Comment:
we add a global handler so we can accept drops on any page and essentially do the share flow